### DISCUSSION TITLE
Vim not starting from PuDB, but nano works

### DISCUSSION THREAD

**Discussion by yogi81** [inducer/pudb#630] [General]
When I hit Ctrl+E in PuDB it starts a currently set up Editor.

With the EDITOR environment variable the user can select the editor he prefers.

Per default nano is opened and it works as expected.

When I set EDITOR=vim then I see following:
1. PuDB quickly disappears (probably it tries to start VIM)
2. Some error message is quickly printed. But I can not read it because 
3. PuDB appears quickly again

This happens with all versions of PuDB.

What can be the reason for this? How to debug PuDB or how can I at least quickly see the error message?

Thanks a lot for a great debugger!

**Comment by inducer** [comment]
I'm really not sure. I also have `$EDITOR` set to `vim`, and things work as expected. You may be able to see the error message if you hit `o`.

**Reply by yogi81** [reply]
Thanks for the answer!

Yes, when I hit 'o' PuDB shows temrinal, but then everything hangs and I do not see the error nor it is clear to me how to exit this mode.

I have to kill python and restart.

**Reply by yogi81** [reply]
Is there a way to log all of the outputs to a file ?

**Reply by yogi81** [reply]
Ok, I will use high speed camera of my smartphone to capture the output 🤦

**Reply by yogi81** [reply]
Ok, there is no error output. Vim just exits. And before that just an information about $PYTHONPATH is shortly shown.

**Reply by yogi81** [reply]
I will check my vim extensions. Something tells me that it is could be YCM extension...
